cdc/sink/metrics/metrics.go

```diff
@@ -38,6 +41,16 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(0.002 /* 2 ms */, 2, 18),
 		}, []string{"changefeed", "type"}) // type is for `sinkType`
 
+	// RowSizeHistogram records the row size of events.
+	RowSizeHistogram = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: "ticdc",
+			Subsystem: "sink",
+			Name:      "received_row_changed_event_size",
+			Help:      "The size of all received row changed events (in bytes)",
+			Buckets:   prometheus.LinearBuckets(rowSizeLowBound, rowSizeLowBound, 10),
```

Comment:
```suggestion
			Buckets:   prometheus.ExponentialBuckets(rowSizeLowBound, 2, 10),
```

So that we can observe a larger range from 128KB to 128MB for some extreme cases.